plugins/multi-cloudinary/README.md

```diff
@@ -0,0 +1,100 @@
+# Builder.io Cloudinary Plugin
+
+This plugin allows you to access your Cloudinary image content in the Builder.io's Visual Editor using the official Cloudinary Media Library widget.
+
+<img src="https://imgur.com/vpNzMud.gif" alt="Plugin example">
+
+## Installation
+To install the plugin navigate to the [Integrations](https://builder.io/app/integrations) tab in your dashboard and click **Enable** for the Cloudinary plugin.
```

Comment:
Might need to change some of these

Reply:
Added plugin instructions specific to multi image selector and a loom showing its usage.